test/controllers/AddItemMoreInformationControllerSpec.scala

```diff
@@ -184,4 +185,138 @@ class AddItemMoreInformationControllerSpec extends SpecBase
       }
     }
   }
+
+  lazy val addItemSealsInformationRoute = routes.AddItemMoreInformationController.loadItemSealsInformation(testErn, testArc, idx, NormalMode).url
+  lazy val addItemSealsInformationSubmitAction = routes.AddItemMoreInformationController.submitItemSealsInformation(testErn, testArc, idx, NormalMode)
+
+  Seq(
+    (AddItemSealsInformationPage(idx), ItemSealsInformationPage(idx), addItemSealsInformationRoute, addItemSealsInformationSubmitAction)
+  ) foreach { case (yesNoPage, infoPage, url, submitAction) =>
```

Comment:
This just needs to be added to the loop above. Rather than duplicating the entire tests